gems/spree_auth_devise/GHSA-8xfw-5q82-3652.yml

```diff
@@ -0,0 +1,33 @@
+---
+gem: spree_auth_devise
+ghsa: 8xfw-5q82-3652
+url: https://github.com/solidusio/solidus_auth_devise/security/advisories/GHSA-xm34-v85h-9pg2
```

Comment:
This advisory is for `solidus_auth_devise`, but the gem name is `spree_auth_devise`.

Reply:
Solidus is a popular fork of Spree, so they share a lot of dependencies

Reply:
(Already included in gems/spree_auth_devise/CVE-2021-41275.yml)